src/core/client/ui/components/icons/FlairNotFoundIcon.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FunctionComponent } from "react";
+
+const FlairNotFoundIcon: FunctionComponent = () => {
+  // https://www.streamlinehq.com/icons/streamline-regular/files-folders/image-files/image-file-warning
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="-0.75 -0.75 48 48"
+    >
+      <path
+        stroke="#000000"
+        d="M33.421875 40.6875a0.7265625 0.7265625 0 0 1 0 -1.453125"
+        strokeWidth="1.5"
+      ></path>
+      <path
+        stroke="#000000"
+        d="M33.421875 40.6875a0.7265625 0.7265625 0 0 0 0 -1.453125"
+        strokeWidth="1.5"
```

Comment:
We want to not include `strokeWidth` here so that we can use css to style.

Reply:
Fixed!